RayLin.md

```diff
@@ -1,12 +1,43 @@
-# 07/18 ~ 07/22
+# 07/25 ~ 07/29
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - UI discussion with Peko
-  - Bug 681553 - [meta] Make HTML5 \<audio\> controls less of a hack
-    - No clear goals for now. Need define what we want controls optimized for audio, and UX should do that defining.
-    - De-prioritized. Save this bug to after visual refresh.
+    - Prefer design 01
+    - Consider special cases:
+      - Play audio with \<video\> (no video frame presents)
+      - Video without audio track
+      - Extreme small size video
+      - Error message refresh
   - Bug 1222273 - Convert HTML \<video\>/\<audio\> controls to be (mostly?) HTML instead of XUL
-    - WIP
-    - All but \<stack\> tag replaced with HTML.
-    - Adjust position: removing XUL specific style
+    - Struggle with mobile UI and XBL inheritance problem.
+    - Conversion progress:
+      - Desktop:
+        - Mostly XUL converted, except \<xul:stack\> and \<xul:scale\>
```

Comment:
What I meant to say is, there are good findings and progress that is worthy of document on the bug itself, not just the weekly here. More people would likely be able to read about it and comment on it (if they could help).